src/solver/optimisation/opt_fonctions.h

```diff
@@ -30,6 +30,7 @@
 #include "../config.h"
 #include "opt_structure_probleme_a_resoudre.h"
 #include "../simulation/sim_structure_donnees.h"
+#include "../optimisation/adequacy_patch.h"
```

Comment:
If we don't add this include, it does not compile ?

Reply:
No the HOURLY_CSR_PROBLEM class is declared in this header

Reply:
ok, thanks